### PR TITLE
Fix Static Analysis errors

### PR DESCRIPTION
**Pull request by AbdiTolesa**
Quoting Mike,

> @AbdiTolesa It also looks like the workflows are failing. Unrelated to your current updates, so it might be better to solve this in a separate PR, but could you help with getting them passing again?
> 
> Thank you!

### CODE REVIEW

File: models/FrmLocImport.php [hunk 1]
```diff
@@ -148,7 +148,7 @@ private static function import_cities_with_counties_csv( $opts ) {
 		if ( $opts['cities'] < self::$cities_total ) {
 			// Get city form ID
 			$form_id = FrmForm::get_id_by_key( 'frm_loc_list' );
-			if ( ! $form_id ) {
+			if ( (bool) $form_id ) {
```
Comment (Crabcyborg) [inline models/FrmLocImport.php:151]: It looks like this condition was flipped.

Also, what true is this? We probably don't need to convert everything to bool. This is a rule I think we're fine ignoring. It doesn't really make the code look better.

Comment (AbdiTolesa) [reply models/FrmLocImport.php]: Thanks @Crabcyborg! I also ignored another rule that is triggered for every `is_callable()` check.

File: us_locations.php [hunk 2]
```diff
@@ -19,11 +19,11 @@ function frm_loc_forms_autoloader( $class_name ) {
 	$path = __DIR__;
 
 	// Only load Frm classes here
-	if ( ! preg_match( '/^FrmLoc.+$/', $class_name ) ) {
+	if ( 1 !== preg_match( '/^FrmLoc.+$/', $class_name ) ) {
 		return;
 	}
 
-	if ( preg_match( '/^.+Controller$/', $class_name ) ) {
+	if ( 1 === preg_match( '/^.+Controller$/', $class_name ) ) {
```
Comment (Crabcyborg) [inline us_locations.php:26]: Can we change a few more back? These checks would be more consistent if we just used the `! preg_match`/`preg_match` checks.

Comment (AbdiTolesa) [reply us_locations.php:26]: I did that change to fix a static analysis error thrown when non boolean value is passed to the check.

**Review by Crabcyborg — APPROVED** [review]
Thanks @AbdiTolesa!

I think this is good to merge.

🚀

**Pull request merged**